vue-frontend/src/pages/portfolio/index.vue

```diff
@@ -6,19 +6,34 @@
       </template>
     </metainfo>
     <Header />
-    <Portfolio />
-    <CTA />
+    <CaseStudy ref="newPortfolio" />
+    <UserReview ref="userReview" class="sm:tw-top-[40px]" />
```

Comment:
The top and bottom space of this section is too much. Check it in all devices and make it same as other sections for all devices